junitparser/cli.py

```diff
@@ -16,6 +16,20 @@ def merge(paths, output):
     return 0
 
 
+def verify(paths):
```

Comment:
would a test calling this method be useful?

Reply:
Yes, it would indeed. I've added the test.